test/__snapshots__/cpa.test.js.snap

```diff
@@ -38,3 +38,37 @@ exports[`should print no matches found 1`] = `
 "No matches found for nomatch.js
 "
 `;
+
```

Comment:
didn't understand ^ .The modules issue is caused because the `sourceType` is not set to `module` right?